src/main/java/net/morbz/osmonaut/osm/Entity.java

```diff
@@ -90,10 +91,7 @@ public boolean equals(Object obj) {
 		if (entity.getEntityType() != getEntityType()) {
```

Comment:
is there a reason why you don't use intelliJ default ways to generate equals and hashCode methods?

Reply:
No, I simply followed the way, already existing methods were implemented. I changed it for the hashCode methods (I used the Objects.hashCode() method instead of generating it with IntelliJ).
I would suggest leaving the equals methods as they are, because they are more detailled.